plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -2069,6 +2083,84 @@ public void testSplitPruningForFilterOnPartitionColumn()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testAllAvailableTypes()
```

Comment:
In this class, we have dedicated methods for every (?) supported type.
Do we need this additional test?
are any type-specific methods missing?

Reply:
> In this class, we have dedicated methods for every (?) supported type.
Do we? For example I don't see a method that checks booleans, of course if i search through the code I will see columns with boolean type used but those are not dedicated methods. Same goes for integers, bigints, varchars etc. Of course there are methods that check more complex behaviour of specific types but explicitly the simplest.
My idea was to have a simple test for all available types in one place. I can drop it if you think this is wrong

Reply:
indeed, the coverage is erratic. let's have a "all types" test then,